examples/time_series/goes_xrs_nrt_data.py

```diff
@@ -64,9 +73,30 @@
 goes_ts = ts.TimeSeries(goes_data, meta, units, source="xrs")
 
 ###############################################################################
-# Finally, we can plot the timeseries.
+# NOAA also provides the past 7 days of flare information which we can also parse.
+
+flare_events = Table.read("https://services.swpc.noaa.gov/json/goes/primary/xray-flares-7-day.json", format='pandas.json')
+
+###############################################################################
+# Finally, we can plot the timeseries and overlay all the flares that occurred.
+# We will do this only for the last day otherwise the plot may be too busy.
 
 fig, ax = plt.subplots()
 goes_ts.plot(axes=ax)
-
+for this_flare in flare_events:
+    start_time, peak_time, end_time = parse_time(
+        [this_flare["begin_time"], this_flare["max_time"], this_flare["end_time"]]
+    )
+    if start_time > (goes_ts.time[-1] - TimeDelta(1 * u.day)):
+        ax.axvline(peak_time.datetime)
+        ax.axvspan(
+            start_time.datetime,
+            end_time.datetime,
+            alpha=0.2,
+            label=f'{peak_time} {this_flare["max_class"]}',
```

Comment:
```suggestion
            label=f'{peak_time} {max_class}',
```
Because I changed the unpacking approach